web/data/data.go

```diff
@@ -466,6 +403,22 @@ func Routes(router *echo.Group) {
 	router.GET("/", dataAPIWelcome)
 	router.GET("/_all_doctypes", allDoctypes)
 
+	// Accounts are handled specifically to remove the auth fields
+	{
+		accountGroup := router.Group("/"+consts.Accounts, accountDoctype)
```

Comment:
Can you add some tests? I had some surprises with echo router while working on the sharing and I'd prefer to be sure that all the routes work as intented